sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -45,7 +45,7 @@ export interface AspectConfidenceScoreLabel {
 // @public
 export interface AspectSentiment {
     confidenceScores: AspectConfidenceScoreLabel;
-    sentiment: SentenceAspectSentiment;
+    sentiment: TokenSentimentValue;
```

Comment:
Did we confirm whether or not we're going to take this type name from the swagger instead of using the two separate type names that we ran through Architecture review?

Reply:
@bterlson the service team decided to consolidate the two sentiment types into one called  `TokenSentimentValue` and I am following that renaming in this PR. The values of both types are the same and non of them were there in our last release. Do you have a strong opinion about this change?